portfolio/src/main/webapp/script.js

```diff
@@ -17,7 +17,8 @@
  */
 function addRandomGreeting() {
```

Comment:
Similar to above, I recommend renaming variables and functions so they more accurately represent what's happening. In this file, you could update the function name, the greetings variables below, and the comment